src/labels/CustomLabels.labels

```diff
@@ -7487,6 +7487,94 @@ If preferred, you can disable Gift Entry and instead use older Batch Gift Entry
         <shortDescription>Gift Processing configuration error</shortDescription>
         <value>Your org is missing required configuration for Gift Processing. Contact your admin for help.</value>
     </labels>
+    <labels>
+        <fullName>gsApplicationStatus</fullName>
+        <categories>gettingStarted</categories>
```

Comment:
The category in the other PR is different than this. I believe that one has "Getting Started Page". Please be sure that the category used for all custom labels across all PR's matches.